src/openms/source/FORMAT/MSPFile.cpp

```diff
@@ -89,6 +83,9 @@ namespace OpenMS
       throw Exception::FileNotReadable(__FILE__, __LINE__, OPENMS_PRETTY_FUNCTION, filename);
     }
 
+    const std::regex rex(R"(\((.*)\))");
```

Comment:
please add comment or example

Reply:
for a pair of parentheses? :)

Reply:
yes or give it a better variable name so one doesn't need to parse the regex